concourse/book-scan/task.yml

```diff
@@ -0,0 +1,20 @@
+---
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    password: ((dockerhub-password))
+    username: ((dockerhub-username))
+    repository: node
+    tag: 14.15.4-buster
```

Comment:
I couldn't see an easy way to get the rex image from the build-rex job instead, but I'm open to suggestions.